libs/sdk-ui-dashboard/src/presentation/shareDialog/index.tsx

```diff
@@ -0,0 +1,6 @@
+// (C) 2019-2021 GoodData Corporation
+export { DefaultShareDialog } from "./DefaultShareDialog";
+export { DefaultShareDialogInner } from "./DefaultShareDialogInner";
+export { CustomShareDialogComponent, IShareDialogProps, IApplyPayload } from "./types";
+export { ShareDialogPropsProvider, useShareDialogProps } from "./ShareDialogPropsContext";
+export { ShareDialogDashboardWrapper } from "./ShareDialogDashboardWrapper";
```

Comment:
why is the wrapper exported? isn't it more of an internal thing?

Reply:
Because its used here, my motivation was separate dialog logic into component.  Its used in DashboardHeader and there is place where dead kittens could be found for each hook.

Reply:
Sorry link missing https://github.com/ivanmjartan/gooddata-ui-sdk/blob/a53a98741e40bbec91ef6ea3a7030cf02efe604c/libs/sdk-ui-dashboard/src/presentation/dashboard/Dashboard.tsx#L308

Reply:
Component renamed to ShareDialogDashboardHeader

Reply:
And moved from  ShareDialog